### ISSUE TITLE
Web interface: Unable to get a regular preset seed after a modified seed of the same preset

### ISSUE THREAD

**Issue by strotlog** [theonlydude/RandomMetroidSolver#34]
**Steps**:

1. Go to 'Randomizer' on website
2. Put in ROM + SMRAT2020 + SMRAT2020 settings.
3. Open advanced settings and modify to remove fanfares (OR enable boss rando, same bug in the end)
4. Play seed, close tab, etc.
5. Wait 24 hours (optional)
...
6. Go to 'Randomizer' on website
7. Put in ROM. Note that SMRAT2020 and SMRAT2020 are already selected (remembered from before). Advanced settings are shown but there is no indication of which do not match the shown preset.
8. In case of weird settings, re-select SMRAT2020 and SMRAT2020
9. Play seed

tl;dr You want a couple modifications to the preset for one seed, and later you want to do an official seed.

**Expected**:
Settings are reset to SMRAT2020 and you play a vanilla-settings seed at the last step.

**Actual**:
The advanced modification you did in step 3 persists through all steps. You play a surprise boss rando seed, etc.

**Comment by theonlydude** [comment]
Indeed the last parameters used to randomize are stored in the session and are reloaded.
To avoid the issue we could deselect the settings preset when a modification is made to a parameter, but that would require to add hijack code for every parameter, it'll be a mess.
What I could do is to not store the settings preset in the session and always set it to empty.

**Comment by theonlydude** [comment]
it's fixed, it'll be in the next release (and tomorrow on beta).
